Firely.Fhir.Packages.props

```diff
@@ -5,17 +5,17 @@
 		<TargetFrameworks>net6.0;netstandard2.0;</TargetFrameworks>
 		<Nullable>enable</Nullable>
 		<GeneratePackageOnBuild>false</GeneratePackageOnBuild>
-		<FhirNetSdkVersion>5.4.0</FhirNetSdkVersion>
+		<FhirNetSdkVersion>5.5.0</FhirNetSdkVersion>
 	</PropertyGroup>
 	
     <!-- Solution-wide properties for NuGet packaging -->
     <PropertyGroup>
         <LangVersion>9.0</LangVersion>
-        <VersionPrefix>4.2.1</VersionPrefix>
-        <VersionSuffix>alpha</VersionSuffix>
+        <VersionPrefix>5.5.0</VersionPrefix>
```

Comment:
Are you suggesting to allign the packages version with the sdk?

Reply:
I think I do. But that's because it still feels like a part of the SDK, but then separated out to be able to phase it out. But there's a problem: if we need to update the legacty validator separately because of a bug, it needs to wait until a new version of the SDK is released as well. Not unlikely, but a consequence.